### ISSUE TITLE
Strange behavior with hard tabs and column select mode

### ISSUE THREAD

**Issue by caleb** [atg/chocolat-public#927]
When a file has hard tab indents, selecting certain regions of the file has the unexpected result of selecting entire lines.

Given this code:

``` html
 <html>
    <head>
        <meta http-equiv="Content-type" content="text/html; charset=utf-8">
        <title></title>
        <link rel="stylesheet" href="/css/master.css" type="text/css" media="screen" title="no title" charset="utf-8">
    </head>
    ^<body id="body">
        <ul>
            <li>one</li>
            <li>two</li>
            <li>three</li>
            <li>four</li>
            <li>five</li>
        </ul>
    </bod^y>
</html>
```

If I select from the first caret before `<body>` and select to after the "d" in `</body>`, it selects the entire lines of the list items. This screenshot shows more clearly what is going on:

https://dl.dropbox.com/u/473648/Images/column%20select%20with%20hard%20tabs%20in%20chocolat.png

Basically, when you try to select just the first "column" of a tab character, it selects the entire line.
